plugins/event_notification/providers/email/lib/batch/recipients/KEmailNotificationCategoryRecipientEngine.php

```diff
@@ -44,7 +44,25 @@ function getRecipients(array $contentParameters)
 			$userList = KBatchBase::$kClient->user->listAction($userFilter, $userPager);
 			foreach ($userList->objects as $user)
 			{
-				$recipients[$user->email] = $user->firstName. ' ' . $user->lastName;
+				if($user->type == KalturaUserType::USER)
+				{
+					$recipients[$user->email] = $user->firstName. ' ' . $user->lastName;
+				}
+				else if($user->type == KalturaUserType::GROUP)
+				{
+					$groupUserIds = $this->getGroupUserIds($user->id);
```

Comment:
need to check what the maximum batch size that can be sent in one mail, on group engine you can get up to 500, here you can reach much more